src/components/LandingPageMap.tsx

```diff
@@ -4,10 +4,17 @@ import { createEffect, onMount } from "solid-js";
 
 export const LandingPageMap = (props: any) => {
   let map;
-  onMount(() => {
+  onMount(async () => {
+    const response = await fetch("https://demotiles.maplibre.org/style.json");
```

Comment:
Wouldn't it be better to have a style in the demotiles repo with globe turned on?
I think it will be beneficial to both examples and this case.

Reply:
PR here
- https://github.com/maplibre/demotiles/pull/31

Reply:
Approved and merged.

Reply:
updated this